benchmarks/cpp/nvfuser/lstm_cell.cpp

```diff
@@ -1,4 +1,4 @@
-#include <torch/csrc/jit/codegen/cuda/arith.h>
+#include <torch/csrc/jit/codegen/cuda/ops/all_ops.h>
```

Comment:
Should we add a TODO to add LSTM cell ops in composite?

Reply:
What is the signature for this LSTM function?
`cy, hy = lstm(x, cx)`